source/whats-new.txt

```diff
@@ -15,10 +15,11 @@ What's New
    :values: reference
 
 .. meta::
-   :keywords: update, new feature, deprecation, upgrade, driver v2.19, driver v2.20, driver v2.21, driver v2.22, driver v2.23, driver v2.24, driver v2.25
+   :keywords: update, new feature, deprecation, upgrade, driver v2.19, driver v2.20, driver v2.21, driver v2.22, driver v2.23, driver v2.24, driver v2.25, driver v2.26
 
 Learn what's new in:
 
+* :ref:`Version 2.26 <csharp-version-2.26>`
```

Comment:
Question: Why is this csharp-version-2.26 while the other links are version-2.25, 2.24, 2.23, etc? Not a blocker but curious about consistency.

Reply:
In the past, we have not been consistent with including the product name in these anchors, but this is important as we sometimes have cross-repo links and this ensures that there wont be collisions/ambiguous links.

Reply:
Got it, thanks!